### PR TITLE
scanning-operator - STG only

### PR DESCRIPTION
**Pull request by rhdedgar**
This is a temporary deployment to STG until a new, special environment is available for scanning-operator.

This is very much a corner-case in our usual workflow, so this PR can be considered a WIP until it's approved.

### CODE REVIEW

File: deploy/scanning-operator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+reviewers:
+- dedgar
```
Comment (2uasimojo) [inline deploy/scanning-operator/OWNERS:2]:
```suggestion
- rhdedgar
```
(your RH GH username)

**Comment by rhdedgar** [comment]
retest

**Pull request closed without merging**